src/main/java/net/levelz/network/PlayerStatsServerPacket.java

```diff
@@ -67,7 +70,10 @@ else if (stat.equals("smithing"))
         ServerPlayNetworking.registerGlobalReceiver(SEND_TAG_PACKET, (server, player, handler, buffer, sender) -> {
             writeS2CTagPacket(player, buffer.readIdentifier());
         });
-
+        ServerPlayNetworking.registerGlobalReceiver(LEVEL_UP_BUTTON_PACKET, (server, player, handler, buffer, sender) -> {
+            if (player == null) return;
+            ((PlayerSyncAccess) player).addLevelExperience(0);
```

Comment:
add 0 experience?

Reply:
Combined with ServerPlayerEntityMixin.addLevelExperience():Line 62, this can be any value.

Reply:
Why is this necessary?

Reply:
Unless I write a new method, but I don't think it is necessary.
This method must have a parameter, so I pass 0, but actually calculate the experience required for the level inside the method, and check.

Reply:
still don't get why the button packet needs to call the addLevelExperience method